src/zenml/cli/server.py

```diff
@@ -242,6 +244,7 @@ def up(
                 f"'{server.status.url}'. You can connect to it using the "
                 f"'{DEFAULT_USERNAME}' username and an empty password."
             )
+            webbrowser.open(server.status.url)
```

Comment:
What is this fails? Maybe we should have try catch here for known errors

Reply:
I've added a try catch statement here.